docker/smqtk_classifier_service/run_container.cpu.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Simple script for starting the SMQTK classifier service
+#
+
+# Container image to use
+CLASSIFIER_CONTAINER=kitware/smqtk/classifier_service
```

Comment:
This could do the same thing that the IQR "run" scripts do with regards to the image name used, since this name is no longer what is built.